source/data-api.txt

```diff
@@ -53,297 +74,71 @@ The Data API supports two types of endpoints:
-  that you write. You can use custom endpoints to run your app's backend
-  logic or as webhooks that integrate with external services.
+  that you write. You can use custom endpoints to define operations that
+  fit your use case specifically. For example, you could create an
+  endpoint that runs a pre-defined aggregation or that integrates with
+  an external webhook service.
 
   To learn more, including how to create and call endpoints, see
   :ref:`Custom HTTPS Endpoints <custom-endpoints>`.
 
-.. _data-api-supported-environments:
-
-Supported Environments
+How the Data API Works
 ----------------------
 
-You can call the Data API from most HTTP clients. Supported environments include:
-
-- Serverless functions
-- Mobile applications
-- Servers
-- Internet of things devices
-
-You **cannot** call the Data API from a web browser or any other client that
-enforces the :mdn:`CORS protocol <Web/HTTP/CORS>`.
-Atlas provides alternative methods to query MongoDB from the web browser:
-
-- Use the :ref:`Realm Web SDK MongoDB client <web-query-mongodb>`.
-- Configure the :ref:`Atlas GraphQL API <graphql-api>`. Then call the GraphQL API
-  from the browser :ref:`using a GraphQL client, like Apollo <graphql-api>`.
-- Create an :ref:`Atlas Function that queries MongoDB <functions-mongodb>`.
-  Then use the Realm Web SDK to :ref:`call the Function <web-call-a-function>`.
-  
-.. note::
-
-   Atlas App Services does not currently support :atlas:`private endpoints </security-private-endpoint/>`.
-
-.. _data-api-data-formats:
-
-Data Formats
-------------
-
-MongoDB stores data in a format called BSON, which is similar to a JSON
-object in structure but supports additional data types and uses a binary
-encoding. BSON is efficient for computers but is not human readable so
-you can't work with it directly.
-
-Instead, the Data API uses two formats to represent data in requests and
-responses:
-
-- The **JSON** format uses standard types that any tool can parse and
-  understand. However, JSON cannot represent all BSON types so JSON
-  responses may lose type information for some fields. For example, BSON
-  has distinct types for 32-bit integers and 64-bit floats but a JSON
-  response represents both as a ``number``.
-
-- The **EJSON** format, short for :manual:`MongoDB Extended JSON
-  </reference/mongodb-extended-json/>`, is a superset of standard JSON
-  that uses structured fields to represent BSON data that don't have
-  corresponding JSON types. This fully represents your data but requires
-  your client to understand how to work with EJSON.
+Data API requests may resemble traditional database operations, like
+``find`` or ``insertOne``, but the Data API is not a direct connection
+to your database. Instead, the Data API adds additional authentication,
+authorization, and correctness checks to ensure that your data is only
+accessed or modified in the ways you allow. This allows you to safely
+access data in Atlas from unsafe clients like web apps.
```

Comment:
to call web apps 'unsafe' rings a little funky to me.

i wonder if there's a better word that we could use here. perhaps smthn like: 

```suggestion
access data in Atlas from potentially vulnerable clients like web apps.
```

(got that suggestion w help of chat gpt 😉)